packages/components/text/src/text.tsx

```diff
@@ -208,20 +217,22 @@ export type TDetailProps = {
     | 'positive'
     | 'negative'
     | 'warning'
-    | 'inverted';
+    | 'inverted'
+    | 'critical';
   truncate?: boolean;
 } & TBasicTextProps &
   TBasicHeadlineProps;
 
-const Detail = (props: TDetailProps) => {
+const Detail = (props: TDetailProps & { isNewTheme?: boolean }) => {
   warnIfMissingTitle(props, 'TextDetail');
   warnIfMissingContent(props, 'TextDetail');
+  const { isNewTheme } = useTheme();
```

Comment:
It seems we don't need the `useTheme` hook here.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2370/commits/e14b989ede6e76d3f24cfce13e50d819f85761ac
Thanks!